pages/features.html

```diff
@@ -6,104 +6,134 @@
+      </p>
+    </div>
+
+  <div style="clear: both;"></div>
+
+    <h1>{% trans "Sophisticated Music Library" %}</h1>
     <p>
-      {% trans "The Mixxx music library was designed from the ground up for DJs. Powered by a high-performance database, accessing and organizing your music is easy and intuitive." %}
+      {% trans "Stay organized and find the music you're looking for the moment you need it. Powered by a high-performance database, navigation and searching your collection is intuitive and fast." %}
     </p>
 
     <div style="clear: both;"></div>
 
     <div class="halfbox_left">
-      <H2><img src="{% static '/static/images/ic_library_itunes.png' %}" class="feature_icon">{% trans "Organization" %}</H2>
+      <h2><img src="{% static '/static/images/ic_library_itunes.png' %}" class="feature_icon">{% trans "Organization" %}</h2>
       <p class="feature_indent">
         <b>{% trans "Crates and Playlists" %}</b><br>
-        {% trans "Sort your music the way you want." %}
+        {% trans "Sort your music the way you want. Use playlists to arrange plans for a set and unordered crates to build your own categorization system." %}
       </p>
       <p class="feature_indent">
-        <b>{% trans "Disk Browsing" %}</b><br>
-        {% trans "Need to find music outside your library? Browse and load tunes from any hard disk." %}
+        <b>{% trans "Search &amp; Sort" %}</b><br>
+        {% trans "Filter your collection by specific tags to narrow your choices or just start typing to find the track you're thinking of. Sort the library table hierarchically with multiple tags at a time." %}
       </p>
       <p class="feature_indent">
         <b>{% trans "iTunes and Traktor Library Integration" %}</b><br>
         {% trans "Drop fresh tracks from your iTunes or Traktor library right into your mix." %}
       </p>
     </div>
-
     <div class="halfbox_right">
       <img width="362px" src="{% static '/static/images/feature_library-2.0.png' %}" style="padding-top: 30px;">
     </div>
 
     <div style="clear: both;"></div>
 
     <div class="halfbox_left">
-      <H2><img src="{% static '/static/images/ic_library_prepare.png' %}" class="feature_icon">{% trans "BPM Detection and Sync" %}</H2>
-      <p class="feature_indent">{% trans "Newly upgraded BPM detection library does an even better job finding the beat in even the toughest rhythms.  Instantly sync the tempo of four songs for seamless beatmixing. Each waveform view marks each beat to help guide your mixing." %}</p>
+      <h2><img src="{% static '/static/images/ic_library_prepare.png' %}" class="feature_icon">{% trans "BPM &amp; Key Detection" %}</h2>
+      <p class="feature_indent">{% trans "Upgraded BPM detection library does an even better job finding the beat in even the toughest rhythms. New musical key detection provides more information to help you pick the next track." %}</p>
```

Comment:
don't say new... the website gets out of date quickly and having the word "new" is not really important except when we do our specific "what's new in this version" pages.